lib/const.rb

```diff
@@ -286,6 +286,15 @@
 
 CREW_CMAKE_LIBSUFFIX_OPTIONS ||= "#{CREW_CMAKE_OPTIONS} -DLIB_SUFFIX=#{CREW_LIB_SUFFIX}"
 
+GEM_ARCH ||= case ARCH
+             when 'x86_64'
+               'x86_64-linux'
+             when 'i686'
+               'x86-linux'
+             when 'aarch64', 'armv7l'
+               'armv8l-linux-eabihf'
```

Comment:
This is not 32-bit?

Reply:
I thought the `7` meant 32-bit and the `8` meant 64-bit?  I was under the impression the `l` meant low power.

Reply:
Were the gem binaries tested in arm containers and confirmed working?

Reply:
Yes. They're working in my arm containers.

Feel free to test on your hardware to confirm.

Reply:
Armv7l and armv8l are both 32-bit.

Arm64 and aarch64 are the 64-bit modes.